### DISCUSSION TITLE
Github Support doesn't reply for 3 weeks<title>

### DISCUSSION THREAD

**Discussion by Jaffies** [community/community#41405] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I've created a ticket #1891670 17 days ago, and there's no any reply.
Responce time for github support is said to be 24 hous.
How can i solve that issue?

**Comment by jfullstackdev** [comment]
they too are triaging issues,

good you posted it here, if one GitHub Staff see this, they might accelerate this request
